plugins/colorschemes/base16/default.nix

```diff
@@ -24,7 +29,7 @@ helpers.vim-plugin.mkVimPlugin config {
     let
```

Comment:
I figured we can keep these renames for now. Should I bump the date in the TODO, e.g. for removal after 24.11?

Reply:
yes, let's do that.